lib/request.js

```diff
@@ -171,6 +172,7 @@ Object.defineProperties(Request.prototype, {
       const routeLimit = context._parserOptions.limit
       const serverLimit = context.server.initialConfig.bodyLimit
       const version = context.server.hasConstraintStrategy('version') ? this.raw.headers['accept-version'] : undefined
+      const { handler, config } = context
```

Comment:
You are taking the reference here and `config` is a object.
When you deep frozen the object, altering `config` in else where will be break.

Reply:
pal